recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -428,8 +428,15 @@ install:
               echo "{\"Metadata\":{\"previousNriBundleDetected\":\"No\"}}" | tee -a {{.NR_CLI_OUTPUT}} > /dev/null
             fi
 
-            $SUDO helm repo add newrelic https://helm-charts.newrelic.com
-            $SUDO helm repo update
+            if [[ $($SUDO helm repo add newrelic https://helm-charts.newrelic.com) ]]; then
+            else
+              exit 131
+            fi
+
+            if [[ $($SUDO helm repo update) ]] ; then
+            else
+              exit 131
```

Comment:
I would suggest adding an echo "ERROR = $? from  $SUDO helm repo update" on the failure path so that we can understand the reason for the failure.

Reply:
Same reasoning for choosing to not set a custom echo as I mentioned in the comment above.